Rover/GCS_Mavlink.cpp

```diff
@@ -342,6 +342,13 @@ bool GCS_MAVLINK_Rover::try_send_message(enum ap_message id)
         break;
     }
 
+    case MSG_AIS_VESSEL: {
+#if HAL_AIS_ENABLED
+        rover.g2.ais.send(chan);
```

Comment:
```suggestion
        rover.g2.ais.send(*this);
```

Reply:
I don't understand, this change.

Reply:
I was suggesting passing the link object through.  There are convenient methods on the object that avoid going through global-static functions to get info and whatnot (which sometimes take that integer and turn it back into the object and then call a method on it!).

Searching for `*this` in `GCS_Common.cpp` will find lots of places where we do that already.

The one call that uses it would be something like `link->send(MAVLINK_MSG_ID_AIS_VESSEL, (char*)_list[_send_index].info)`

That being said - not really a blocker, so I'm resolving this as a "meh".